src/Sylius/Component/Core/Handler/CodeChangeHandlerInterface.php

```diff
@@ -9,14 +9,14 @@
  * file that was distributed with this source code.
  */
 
-namespace Sylius\Component\Core\Locale\Handler;
+namespace Sylius\Component\Core\Handler;
 
 use Sylius\Component\Core\Exception\HandleException;
 
 /**
  * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
  */
-interface LocaleChangeHandlerInterface
+interface CodeChangeHandlerInterface
```

Comment:
On one hand 👍 , on the other there is no CompositeLocaleChangeHandler, but one generic which handles actual code change as the `->handle($code)` method states. 
With two different interfaces, one for locale, and one for currency, there would be two composite classes, which are virtually the same, only different by interface, whereas now the distinction for locale/currency is container in the service definition.